### PR TITLE
Improve `state` related logs to use a more uniform format

### PR DESCRIPTION
**Pull request by bkchr**
The logging before wasn't that uniform and not that great to read/parse.
Now we are using a uniform format for all the logs. Besides these
changes, there are some minor changes around the code that calls the
state machine.

### CODE REVIEW

**Comment by arkpar** [comment]
Looks good. My only gripe is that `ext_id` is not hex-formatted. Maybe switch it to something like  `ext_id = HexDisplay::from(&self.id.to_le_bytes())`?

**Review by arkpar — APPROVED** [review]

**Comment by bkchr** [comment]
> Looks good. My only gripe is that `ext_id` is not hex-formatted. Maybe switch it to something like `ext_id = HexDisplay::from(&self.id.to_le_bytes())`?

I have done this by intention. There was already some logs that switched to the non-hex format. Why do you think that hex formatting is better? (Because the id could be something big?)

**Comment by arkpar** [comment]
Id is a random number so it will have random width when decimal-formatted.

**Review by andresilva — APPROVED** [review]

File: primitives/state-machine/src/ext.rs [hunk 1]
```diff
@@ -402,7 +412,7 @@ where
 		trace!(
 			target: "state",
 			method = "Put",
-			ext_id = self.id,
+			ext_id = %HexDisplay::from(&self.id.to_le_bytes()),
```
Comment (bkchr) [inline primitives/state-machine/src/ext.rs:415]: CC @emostov

**Pull request merged**